cdap-api/src/main/java/co/cask/cdap/api/data/format/StructuredRecord.java

```diff
@@ -56,6 +56,10 @@ public Schema getSchema() {
     return (T) fields.get(fieldName);
   }
 
+  public Map<String, Object> getFields() {
+    return fields;
```

Comment:
The map is not immutable. Seems dangerous to expose it. If really needed, make the map immutable in constructor.